src/domain/event/eventQueryString.util.ts

```diff
@@ -0,0 +1,30 @@
+import { ROUTES } from '../app/routes/constants';
+
+export type ReturnParams = {
+  returnPath: string;
+  remainingQueryString?: string;
+};
+
+/**
+ * Extracts latest return path from queryString. For example on:
+ * http://localhost:3000/fi/event/kulke:53397?returnPath=%2Fevents&returnPath=%2Fevent%2Fhelsinki%3Aaf3pnza3zi
+ * latest return path is in the last returnPath param on queryString : %2Fevent%2Fhelsinki%3Aaf3pnza3zi
+ * @param queryString
```

Comment:
does this line add anything useful, could it be deleted?

Reply:
> does this line add anything useful, could it be deleted?

An URL example is a good addition. It helped me to understand whats the input to this function.

Reply:
I was only referring to line 12:

 * @param queryString

Reply:
Example is good 👍

Reply:
Removed. IntelliJ added it automatically.